testing/trino-product-tests-launcher/src/main/java/io/trino/tests/product/launcher/env/common/Standard.java

```diff
@@ -184,6 +185,61 @@ else if (logicalName.startsWith(WORKER_NTH)) {
         enableTrinoJavaDebugger(dockerContainer, logicalName, debugPort);
     }
 
+    private static void enableTrinoJmxRmi(DockerContainer dockerContainer)
+    {
+        String logicalName = dockerContainer.getLogicalName();
+
+        int jmxPort;
+        if (logicalName.equals(COORDINATOR)) {
+            jmxPort = 6005;
+        }
+        else if (logicalName.equals(WORKER)) {
+            jmxPort = 6009;
+        }
+        else if (logicalName.startsWith(WORKER_NTH)) {
+            int workerNumber = parseInt(logicalName.substring(WORKER_NTH.length()));
+            jmxPort = 6008 + workerNumber;
```

Comment:
why not contiguous?

Reply:
Good question. I've used the same base as in the exposing debugger part, shifted by 1000